projects/ion/src/lib/no-data/no-data.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+import { IconType } from '../core/types';
+
+@Component({
+  selector: 'ion-no-data',
+  templateUrl: './no-data.component.html',
+  styleUrls: ['./no-data.component.scss'],
+})
+export class IonNoDataComponent {
+  @Input() iconType?: IconType;
```

Comment:
```suggestion
  @Input() iconType?: IconType;
```

Add default icon here